Common/Source/Devices/devLXNano3.cpp

```diff
@@ -190,7 +190,7 @@ BOOL DevLXNanoIII::Install(PDeviceDescriptor_t d) {
   d->IsGPSSource  = GetTrue;
   d->IsBaroSource = GetTrue;
   d->Config       = Config;
-  d->DirectLink   = Nano3_DirectLink;
+  d->DirectLink   = NULL; // Nano3_DirectLink;
```

Comment:
useless, already done by `DeviceDescriptor_t::InitStruct(int)`

Reply:
Yes I know, but
Nano3_DirectLink
is also still implemented. For the next step I'm tinking to make it configureable in the Nano3 Config Dialog (and maybe Messagebox Direct Link Yes/No Questions) to reactivate this functionality for the full featured driver.
At the moment I'm in doubt is this function is realy used.